### ISSUE TITLE
missing data discuss 

### ISSUE THREAD

**Issue by quinn-dougherty** [labs12-real-estate/ds-scripts#9]
i watched [this](https://youtu.be/2gkw2T5jAfo) again last night. 

I'm first gonna write a sort of logger that captures "for each feature: nan rate: xx%, correlative/collinear features: A,B,C, ; missing regime estimate: (MCAR, MAR, or MNAR) (this will be silly, because i think it's almost entirely MAR); impute strategy recommendation: (const, KNN, MICE, etc) ". 

obviously missing regime estimate and impute strategy recommendation is _hard to write_ in the logger, so i'll start with a trivial one that just captures nan rate by percentage and 

it appears that the only real issue w `fancyimpute` is that you need to cast between numpy and pandas, and if MICE is correct we shouldn't let that stop us! https://pypi.org/project/fancyimpute/ (fancy impute is like the only MICE implementation available in python)

**Comment by quinn-dougherty** [comment]
TLDR: feel free to do whatever imputation you'd normally do--- the minimum we should _definitely_ hold ourselves too (because it's trivially easy) is **documenting and reporting** the way we found the data. 

For reference, some slides from the talk: 
![Screenshot_20190426-002728_YouTube](https://user-images.githubusercontent.com/39039420/56820483-02fbb700-681a-11e9-82e3-6dd7c4e973dd.jpg)
![Screenshot_20190426-002721_YouTube](https://user-images.githubusercontent.com/39039420/56820485-03944d80-681a-11e9-8c38-cd4d63e66645.jpg)
![Screenshot_20190426-001002_YouTube](https://user-images.githubusercontent.com/39039420/56820486-03944d80-681a-11e9-8e13-f4a5f9d7df68.jpg)

**Comment by quinn-dougherty** [comment]
this is initialized https://github.com/labs12-real-estate/ds-scripts/tree/quinn-dougherty/load/logna.py , need to figure out collinearity inferrer.
